phpzlc/phpzlc/1.0/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+    "bundles": {
+        "PHPZlc\\PHPZlc\\Bundle\\PHPZlcBundle": ["all"]
+    },
+    "copy-from-package": {
+        "Doctrine/ORM/Rewrite/Templates/Repository.tpl.php": "vendor/symfony/maker-bundle/src/Resources/skeleton/doctrine/Repository.tpl.php",
+        "Doctrine/ORM/Rewrite/Hydration/ObjectHydrator.php": "vendor/doctrine/orm/lib/Doctrine/ORM/Internal/Hydration/ObjectHydrator.php",
+        "Doctrine/ORM/Rewrite/MakeEntityRegenerate/ClassSourceManipulator.php": "vendor/symfony/maker-bundle/src/Util/ClassSourceManipulator.php",
+        "Doctrine/ORM/Rewrite/MakeEntityRegenerate/EntityRegenerator.php": "vendor/symfony/maker-bundle/src/Doctrine/EntityRegenerator.php",
+        "Bundle/phpzlc.yaml": "config/packages/phpzlc.yml",
+        "PROJECT_README.md":"README.md"
```

Comment:
Why this file?

Reply:
PROJECT_README.md ? I need to tell project users how to use my project, and this file is a template so I don't have to write it every time